src/redux/greeting/greeting.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-param-reassign */
+// Imports
```

Comment:
- [ ] Linters used to highlight errors in our code so that we can have a standard to follow as developers. Kindly uncomment the linters here and fix the error it will bring.